src/main/kotlin/com/example/demo/config/SecurityConfig.kt

```diff
@@ -0,0 +1,37 @@
+package com.example.demo.config
+
+import com.example.demo.domain.Users
+import com.example.demo.repository.UsersRepository
+import com.example.demo.service.UsersDetailsService
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder
+import org.springframework.security.config.annotation.web.builders.HttpSecurity
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder
+
+@EnableWebSecurity
+class SecurityConfig @Autowired constructor(private val usersRepository: UsersRepository) :
```

Comment:
コンストラクタインジェクションに `@AutoWired` は不要だったような……？
記憶違いだったらすみません

```suggestion
class SecurityConfig(private val usersRepository: UsersRepository) :
```

Reply:
調べてみたところ、Spring 4.3からは単一のコンストラクタの場合 @Autowired は省略可能らしいです。消しておきます！

すみません、ログイン用のエンドポイントというのが調べてもよくわからなかったのですが、例えばどういったものでしょうか？

Reply:
> すみません、ログイン用のエンドポイントというのが調べてもよくわからなかったのですが、例えばどういったものでしょうか？

コントローラーを探してもエンドポイントが見つからなかった(PostMappingがなかった)ので質問したのですが、SecurityConfigで設定されている感じですよね？

ちょっと今手元で確認できないので、ログイン時の画面もスクリーンショットで添付してもらえると助かります:pray::pray::pray:

Reply:
エンドポイントの意味、わかりました！
ログイン画面はこのようになっています（localhost:8080/login）

<img width="601" alt="スクリーンショット 2022-08-25 15 33 44" src="https://user-images.githubusercontent.com/48623999/186593357-6f328a20-dd9a-46a8-a41f-c4afc8b9fe32.png">

Spring Securityが/loginを自動でエンドポイントとして設定してくれているようです。

ログアウト機能も試していたのですが、
こちらも何も設定しなくても/logoutが
エンドポイントになっていました。

Reply:
スクショ・補足ありがとうございます！